mne_bids/commands/mne_bids_raw_to_bids.py

```diff
@@ -35,6 +35,9 @@ def run():
     parser.add_option('--run', dest='run',
                       help='The run number for this dataset.',
                       metavar='run')
+    parser.add_option('--acq', dest='acq',
+                      help='The acquisition parameter.',
+                      metavar='acq')
     parser.add_option('--kind', dest='kind',
```

Comment:
could you update https://github.com/mne-tools/mne-bids/pull/106/files#diff-0b05a5040f501fe3b6bc1f14bb15c31bL20 `parser.set_defaults(kind='meg')` --> `parser.set_defaults(kind='auto')`